Workflow.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'Workflow'
-  s.version      = '0.8.0'
+  s.version      = '0.9.0'
```

Comment:
@timdonnelly Is this the only version number I need to change?

Reply:
yes for iOS and distribution for CocoaPods.

Reply:
Thanks!